deploy/001_deploy_swap.ts

```diff
@@ -0,0 +1,54 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+import {
+  getProxyAdminAddress,
+  getSideWrappedBtcAddress,
+} from "../utils/address";
+import { ethers } from "hardhat";
+import { DeployResult } from "hardhat-deploy/dist/types";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+
+  const { deployer } = await getNamedAccounts();
+  const proxyAdminAddress = await getProxyAdminAddress(hre);
+  const sideWrappedBtcAddress = await getSideWrappedBtcAddress(hre);
+
+  const factorySwapRBTC = await ethers.getContractFactory(
+    "SwapRBTC",
+    proxyAdminAddress
```

Comment:
proxyAdmin is the owner of the proxy contract not the SwapRBTC instance

Reply:
this is kind of a workaround to the proxyAdmin be the sender when I tried to use the upgrades from openzeppelin, but using the deploy from hardhat worked better 

I will removed, since it`s not being used

Reply:
done at https://github.com/rsksmart/swap-rbtc/pull/1/commits/a2a80a577dbdadb9cb65c36f34ffa5892d21e50b